js/guests.js

```diff
@@ -0,0 +1,130 @@
+    Worship Assembly in Atlanta, Georgia.`,
+  },
+
+  {
+    underlay: {
+      imageUrl: './images/rubik.png',
+      imageAlt: 'Rubik',
+    },
+    image: {
+      imageUrl: './images/Jekalyn.jpg',
+      imageAlt: 'Jekalyn Carr',
+    },
+    name: 'Jekalyn Carr',
+    shortDescription: 'Gospel Artiste',
+    profile: `Jekalyn Carr is a GRAMMY® nominee and GMA Dove Award-winning independent recording artist. Carr is a
+    speaker, entrepreneur, actress, and author.As a top selling recording artist, Carr has earned eight
+    Billboard #1’s before the age of 21, including singles, “You’re Bigger” and “You Will Win”.`,
+  },
+
+  {
+    underlay: {
+      imageUrl: './images/rubik.png',
+      imageAlt: 'Rubik',
+    },
+    image: {
+      imageUrl: './images/jon.jpg',
+      imageAlt: 'Jonathan Mcreynolds',
+    },
+    name: 'Jonathan Mcreynolds',
+    shortDescription: 'Gospel Artiste',
+    profile: `Jonathan McReynolds is a Grammy-winning American gospel musician. He began his music career in 2012
+    with the release of Life Music via Entertainment One Music. He has had two No. 1 Billboard Gospel Albums
+    since with Life Music: Stage Two (2015) and Make Room (2018), eight Stellar Awards, two GMA Dove Awards
+    and five Grammy Award nominations.`,
+  },
+
+  {
+    underlay: {
+      imageUrl: './images/rubik.png',
+      imageAlt: 'Rubik',
+    },
+    image: {
+      imageUrl: './images/mercy.jpg',
+      imageAlt: 'Mercy Chinwo',
+    },
+    name: 'Mercy Chinwo',
+    shortDescription: 'Gospel Artiste',
+    profile: `Mercy Chinwo is a Nigerian gospel musician, singer and songwriter. She won the Nigerian Idol Season 2
+    in 2012. In 2018,she was awarded the Best Gospel Artiste at the CLIMAX Awards 2018, and in 2020, she
+    emerged as the Gospel Artiste of the year at the AFRIMMA Awards.`,
+  },
+
+  {
+    underlay: {
+      imageUrl: './images/rubik.png',
+      imageAlt: 'Rubik',
+    },
+    image: {
+      imageUrl: './images/Travis.jpg',
+      imageAlt: 'Travis Greene',
+    },
+    name: 'Travis Greene',
+    shortDescription: 'Gospel Artiste',
+    profile: `Travis Greene is an American gospel musician and pastor. Greene's music has charted on the Billboard
+    Top Gospel Albums and the Top Gospel Songs charts and he has been nominated for multiple awards, including
+    5 Grammy Awards and 13 Stellar Awards.`,
+  },
+
+  {
+    underlay: {
+      imageUrl: './images/rubik.png',
+      imageAlt: 'Rubik',
+    },
+    image: {
+      imageUrl: './images/victoria.jpg',
+      imageAlt: 'Victoria Orenze',
+    },
+    name: 'Victoria Orenze',
+    shortDescription: 'Gospel Artiste',
+    profile: `Victoria Orenze was born on the 23rd of March. She is not only a preacher, but fire personified. She
+    believes in the true worship of Jesus Christ and that every Christian should believe in the Almighty God
+    and the spreading of the gospel in every generation.`,
```

Comment:
Nice to work on the changes request and your site looks awesome, however, let us fix this issue.

- [ ] Original Guest
![proje1](https://user-images.githubusercontent.com/55635977/128210005-0ab4ae58-1fc6-43ff-9c30-231c41f7f513.png)

- [ ] Your Guest

![proje22](https://user-images.githubusercontent.com/55635977/128210323-5915dd9c-5de9-4d9b-9780-65f86ecda4a3.png)

On this, there is only one issue to be solved, kindly, use the images that have a high resolution or find another way of correcting that issue for the UX (user experience and interface) purpose. All in all your images look different from the design we have.